syntax = 'proto3';
message BookResponse {
  int32 id = 1;
  string name = 2;
  string author = 3;
  int32 year = 4;
  int32 pages = 5;
}

message ListBooksResponse {
     repeated BookResponse books = 1;
}
